doc/source/whatsnew/v0.25.0.rst

```diff
@@ -475,6 +474,7 @@ Other
 - Removed unused C functions from vendored UltraJSON implementation (:issue:`26198`)
 - Bug in :func:`factorize` when passing an ``ExtensionArray`` with a custom ``na_sentinel`` (:issue:`25696`).
 - Allow :class:`Index` and :class:`RangeIndex` to be passed to numpy ``min`` and ``max`` functions.
+- Allow keyword arguments for callable local reference used in the ``DataFrame.query`` string
```

Comment:
use ``:method:`DataFrame.query` ``; add the issue number (I guess no issue, so this PR number);

move this to indexing section

Reply:
I've moved the entry in the indexing section